Interesting failures on Windows:

```
ERROR: test_force_swmr_mode_off_raises (h5py.tests.hl.test_dataset_swmr.TestDatasetSwmrRead)
Switching SWMR write mode off is only possible by closing the file.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\tests\hl\test_dataset_swmr.py", line 58, in setUp
    self.f = h5py.File(fname, 'r', swmr=True)
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 269, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, swmr=swmr)
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 98, in make_fid
    flags |= h5f.ACC_SWMR_READ
AttributeError: 'module' object has no attribute 'ACC_SWMR_READ'
======================================================================
ERROR: test_force_swmr_mode_on_raises (h5py.tests.hl.test_dataset_swmr.TestDatasetSwmrRead)
Verify when reading a file cannot be forcibly switched to swmr mode.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\tests\hl\test_dataset_swmr.py", line 58, in setUp
    self.f = h5py.File(fname, 'r', swmr=True)
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 269, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, swmr=swmr)
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 98, in make_fid
    flags |= h5f.ACC_SWMR_READ
AttributeError: 'module' object has no attribute 'ACC_SWMR_READ'
======================================================================
ERROR: test_initial_swmr_mode_on (h5py.tests.hl.test_dataset_swmr.TestDatasetSwmrRead)
Verify that the file is initially in SWMR mode
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\tests\hl\test_dataset_swmr.py", line 58, in setUp
    self.f = h5py.File(fname, 'r', swmr=True)
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 269, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, swmr=swmr)
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 98, in make_fid
    flags |= h5f.ACC_SWMR_READ
AttributeError: 'module' object has no attribute 'ACC_SWMR_READ'
======================================================================
ERROR: test_read_data (h5py.tests.hl.test_dataset_swmr.TestDatasetSwmrRead)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\tests\hl\test_dataset_swmr.py", line 58, in setUp
    self.f = h5py.File(fname, 'r', swmr=True)
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 269, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, swmr=swmr)
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 98, in make_fid
    flags |= h5f.ACC_SWMR_READ
AttributeError: 'module' object has no attribute 'ACC_SWMR_READ'
======================================================================
ERROR: test_refresh (h5py.tests.hl.test_dataset_swmr.TestDatasetSwmrRead)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\tests\hl\test_dataset_swmr.py", line 58, in setUp
    self.f = h5py.File(fname, 'r', swmr=True)
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 269, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, swmr=swmr)
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 98, in make_fid
    flags |= h5f.ACC_SWMR_READ
AttributeError: 'module' object has no attribute 'ACC_SWMR_R===== testing package: h5py-2.7.1-py27_2 =====
running run_test.py
EAD'
======================================================================
ERROR: test_extend_dset (h5py.tests.hl.test_dataset_swmr.TestDatasetSwmrWrite)
Extend and flush a SWMR dataset
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\tests\hl\test_dataset_swmr.py", line 127, in test_extend_dset
    self.f.swmr_mode = True
  File "h5py\_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py\_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 225, in swmr_mode
    self.id.start_swmr_write()
AttributeError: 'h5py.h5f.FileID' object has no attribute 'start_swmr_write'
======================================================================
ERROR: test_extend_dset_multiple (h5py.tests.hl.test_dataset_swmr.TestDatasetSwmrWrite)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\tests\hl\test_dataset_swmr.py", line 140, in test_extend_dset_multiple
    self.f.swmr_mode = True
  File "h5py\_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py\_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 225, in swmr_mode
    self.id.start_swmr_write()
AttributeError: 'h5py.h5f.FileID' object has no attribute 'start_swmr_write'
======================================================================
ERROR: test_switch_swmr_mode_off_raises (h5py.tests.hl.test_dataset_swmr.TestDatasetSwmrWrite)
Switching SWMR write mode off is only possible by closing the file.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\tests\hl\test_dataset_swmr.py", line 118, in test_switch_swmr_mode_off_raises
    self.f.swmr_mode = True
  File "h5py\_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py\_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 225, in swmr_mode
    self.id.start_swmr_write()
AttributeError: 'h5py.h5f.FileID' object has no attribute 'start_swmr_write'
======================================================================
ERROR: test_switch_swmr_mode_on (h5py.tests.hl.test_dataset_swmr.TestDatasetSwmrWrite)
Switch to SWMR mode and verify
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\tests\hl\test_dataset_swmr.py", line 111, in test_switch_swmr_mode_on
    self.f.swmr_mode = True
  File "h5py\_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py\_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "C:\bld\h5py_1510157386991\_t_env\lib\site-packages\h5py\_hl\files.py", line 225, in swmr_mode
    self.id.start_swmr_write()
AttributeError: 'h5py.h5f.FileID' object has no attribute 'start_swmr_write'
----------------------------------------------------------------------
Ran 446 tests in 5.365s
FAILED (errors=9, skipped=18, expected failures=6)
```

Maybe related to https://github.com/h5py/h5py/issues/880

--- a/recipe/bld.bat
+++ b/recipe/bld.bat
@@ -1,4 +1,4 @@
-"%PYTHON%" setup.py configure --hdf5="%LIBRARY_PREFIX%"  --hdf5-version=1.8.18
+"%PYTHON%" setup.py configure --hdf5="%LIBRARY_PREFIX%"  --hdf5-version=1.10.1
 if errorlevel 1 exit 1
 
 "%PYTHON%" setup.py install --single-version-externally-managed --record record.txt

